llvm/lib/Target/RISCV/RISCVInstrInfo.cpp

```diff
@@ -2710,6 +2713,45 @@ MachineInstr *RISCVInstrInfo::emitLdStWithAddr(MachineInstr &MemI,
       .setMemRefs(MemI.memoperands())
       .setMIFlags(MemI.getFlags());
 }
+bool RISCVInstrInfo::isPairableLdStInstOpc(unsigned Opc) {
+  switch (Opc) {
+  default:
+    return false;
+  case RISCV::SH:
```

Comment:
This list doesn't match what's supported in `RISCVLoadStoreOpt::tryConvertToLdStPair`

Reply:
That is true. I have added a TODO marker for it in https://github.com/llvm/llvm-project/pull/121394.